.mbedignore

```diff
@@ -1,10 +1,16 @@
+# Ignore ARM GCC toolchain directory
+gcc-*
+
 # Note: Comment out to use display as IIO client (pocket lab), else uses PC as IIO client
 # This also require to set ACTIVE_IIO_CLIENT == IIO_CLIENT_LOCAL in the app_config.h file
-libraries/stm32_lvgl/
-libraries/lvgl/
-libraries/precision-converters-library/pocket_lab/
-libraries/precision-converters-library/fft/
-libraries/CMSIS-DSP/
+libs/stm32_lvgl/
+libs/lvgl/
+libs/precision-converters-library/pocket_lab/
+libs/precision-converters-library/fft/
+libs/CMSIS-DSP/
+libs/no-OS/drivers/axi_core/
```

Comment:
do not add these files here. Add them into .mbedignore file present in the libraries folder